.circleci/config.yml

```diff
@@ -17,6 +17,26 @@ jobs:
       - checkout
       - run: curl -L https://git.io/vp6lP | sh # gometalinter
       - run: PATH=./bin:$PATH gometalinter --deadline=5m --disable-all --enable=gofmt --enable=vet --vendor ./...
+  # FIXME: this is pinned to v3.0.0 because subsequent versions of ct pull in new versions of helm, which are subject
+  #        to this bug: https://github.com/helm/helm/issues/8835, specifically for rbac object names with ':' in them
+  helm-lint:
+    working_directory: /repo
+    docker:
+      - image: quay.io/helmpack/chart-testing:v3.0.0
+    steps:
+      - checkout
+      - run: ct lint
+  # FIXME: instrumenta/helm-conftest does not yet support helm v3, v2 causes tests to fail.
+  #        Switch to it once https://github.com/instrumenta/helm-conftest/pull/8 is merged
+  helm-conftest:
+    working_directory: /repo
+    docker:
+      - image: alpine/helm:3.2.4
+    steps:
+      - run: apk add --update --no-cache git curl bash
+      - run: helm plugin install --debug https://github.com/instrumenta/helm-conftest
+      - checkout
+      - run: helm conftest charts/metallb/ -p charts/metallb/policy/ --fail-on-warn
```

Comment:
Very small nit: This PR seems to mix CircleCI additions and Github Actions. 
Is there a reason we cannot use only CircleCI or Github Actions?

Reply:
Sorry for the late response.  The GitHub Action is a convenient pre-made way to publish the Helm chart in GitHub Pages.  The only reason to use it is to avoid porting that same logic to CircleCI.